pmd-swift/src/main/antlr4/net/sourceforge/pmd/lang/swift/antlr4/Swift.g4

```diff
@@ -1078,6 +1078,13 @@ fragment MultiQuotedTextItem : MultiInterpolatedString
  ;
 fragment MultiInterpolatedString: '\\(' (MultiQuotedTextItem | SingleStringLiteral)* ')';
 
+DashedSingleStringLiteral : '#"' DashedMultiQuotedText? '"#' ;
+fragment DashedMultiQuotedText : DashedMultiQuotedTextItem+ ;
+fragment DashedMultiQuotedTextItem : EscapedCharacter | DashedMultiInterpolatedString
```

Comment:
I'm not sure `EscapedCharacter` should be here. Apparently those dashed string literals are raw strings, so the only escape being processed is `\#(..)` I guess

Looking at this, it looks like `MultiQuotedTextItem` (just above) is missing EscapedCharacter too...